src/formatDate.js

```diff
@@ -8,7 +8,34 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldSeparator = fromFormat.slice(3);
+  const newSeparator = toFormat.slice(3);
+  const splitDate = date.split(oldSeparator);
```

Comment:
The `split` method is not used correctly. The `oldSeparator` is an array but it should be a string. After fixing the `oldSeparator` to be a string, you can use it here correctly.